docs/en/basic/Command-Line-Interface.md

```diff
@@ -404,54 +402,55 @@ data files and their total size for any date, month, or year.
 
-For example, if the Alluxio space is unexpectedly over utilized, `du` can be used to detect which
-folders are taking up the most space.
+For example, if the Alluxio space is unexpectedly overutilized,
+`du` can be used to detect which folders are taking up the most space.
 
 {% include Command-Line-Interface/du.md %}
 
 ### fileInfo
 
-The `fileInfo` command is deprecated since Alluxio version 1.5. Please use `alluxio fs stat <path>`
-command instead.
+The `fileInfo` command is deprecated since Alluxio version 1.5.
+Please use `alluxio fs stat <path>` command instead.
 
-The `fileInfo` command dumps the FileInfo representation of a file to the console. It is primarily
-intended to assist powerusers in debugging their system. Generally viewing the file info in the UI
-will be much easier to understand.
+The `fileInfo` command dumps the FileInfo representation of a file to the console.
+It is primarily intended to assist power users in debugging their system.
+Generally viewing the file info in the UI is much easier to understand.
 
-For example, `fileInfo` can be used to debug the block locations of a file. This is useful when
-trying to achieve locality for compute workloads.
+For example, `fileInfo` can be used to debug the block locations of a file.
+This is useful when trying to achieve locality for compute workloads.
 
 {% include Command-Line-Interface/fileInfo.md %}
 
 ### free
 
-The `free` command sends a request to the master to evict all blocks of a file from the Alluxio
-workers. If the argument to `free` is a directory, it will recursively `free` all files. This
-request is not guaranteed to take effect immediately, as readers may be currently using the blocks
-of the file. `free` will return immediately after the request is acknowledged by the master. Note
-that, files must be persisted already in under storage before being freed, or the `free` command
-will fail; also any pinned files cannot be freed unless `-f` option is specified. The `free` command
-does not delete any data from the under storage system, but only removing the blocks of those files
-in Alluxio space to reclaim space. In addition, metadata will not be affected by this operation,
-meaning the freed file will still show up if an `ls` command is run.
+The `free` command sends a request to the master to evict all blocks of a file from the Alluxio workers.
+If the argument to `free` is a directory, it will recursively `free` all files.
+This request is not guaranteed to take effect immediately,
+as readers may be currently using the blocks of the file.
+`free` will return immediately after the request is acknowledged by the master.
+Note that files must be already persisted in under storage before being freed or the `free` command will fail.
+Any pinned files cannot be freed unless `-f` option is specified. TODO: i have yet to read anything about pinning in the root level docs or advanced section???
```

Comment:
We should add docs for pinning in the "Alluxio Storage Management" doc, I think it was an oversight that it's not currently there